recipe/meta.yaml

```
@@ -6,41 +6,50 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: a0258537e65e7e14ec33a86802f865a667f949bb6c73646d55e42f7c45a052ae
 
 build:
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation  -vv
-  noarch: python
+  number: 1
+  skip: true  # [py<39]
```

https://github.com/serge-sans-paille/beniget/blob/0.4.2.post1/setup.py#L38
```suggestion
  skip: true  # [py<36]
```